resources/assets/app.js

```diff
@@ -5,11 +5,18 @@ import React from 'react';
 import ReactDom from 'react-dom';
 
 import CampaignOverview from './components/CampaignOverview'
+import CampaignInbox from './components/CampaignInbox'
 
 ready(() => {
   const overviewContainer = document.getElementById('overviewContainer');
+  const inboxContainer = document.getElementById('inboxContainer');
+
 
   if (overviewContainer) {
     ReactDom.render(<CampaignOverview {...window.STATE} />, overviewContainer);
   }
+
+  if (inboxContainer) {
```

Comment:
yeah I agree that it's not ideal. I kind of feel like it should be a separate task though? either way I'm going to make an issue for that now so we have a reference/dedicated place to discuss